Add C64 palette

MineRobber9000 opened the pull request:
Since you already have the ZX Spectrum palette, I figured I'd help and add the C64 palette as well.

MineRobber9000 committed:
Add C64 palette

Next commit will include actual palette colors, just gotta get the groundwork down.

MineRobber9000 committed:
Add colors

After a while of mucking around with coding I was able to render this wonderful palette.

Wendelstein7 commented:
Thanks a lot for your submission! 👍 
For keeping consistency with the rest of palCollection, please modify your changes:
- Use camelCase rather than snake_case.
- Use EN-GB spelling. Grey instead of gray.

MineRobber9000 committed:
Fix color names

Wow, I wasn't even internally consistent. `lightRed` vs `light_gray`. Whatever. Camel cased and British-spelling-ized.

MineRobber9000 commented:
Alright, done.

Wendelstein7 commented:
Thank you so much! ❤️
